### PR TITLE
GITHUB-5573: Save btn and attr removing updates

### PR DESCRIPTION
**Pull request by a2xchip**
[//]: <> (<3 Thanks for taking the time to contribute! You're awesome! <3)

[//]: <> (If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/.github/CONTRIBUTING.md)

**Description (for Contributor and Core Developer)**

- Change save button position
- Drop attribute removal confirmation
- Update tests

[//]: <> (What does this Pull Request do? reference the related issue?)

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | OK
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: features/family/remove_attribute_from_a_family.feature [hunk 1]
```diff
@@ -49,7 +48,7 @@ Feature: Remove attribute from a family
       | mobile     | French (France)         | warning  | 1 missing value | 50%   |
     When I am on the "Bags" family page
     And I visit the "Attributes" tab
-    And I remove the "Manufacturer" attribute
+    And I remove the "manufacturer" attribute
     Then I should see the flash message "Attribute successfully removed from the family"
```
Comment (nidup) [inline features/family/remove_attribute_from_a_family.feature]: imho, we should not see anymore this flash message as the attribute is remove when we save the family, and we should save the family in this scenario

Comment (a2xchip) [inline features/family/remove_attribute_from_a_family.feature]: I updated this scenario, for the future, right now it is anyway `skip` tagged one

**Comment by a2xchip** [comment]
Closed in favor of #5793

**Comment by a2xchip** [comment]
Can be needed - reopened

**Comment by a2xchip** [comment]
:tada: :tada: :tada:

**Pull request merged**